release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1984,7 +1995,9 @@ oc create -f $PROJ-app.yaml
 +
 For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1812212[*BZ#1812212*].
 
-* Parsing undefined values results in a not a number (NaN) exception and the *Chart* tooltip shows a box with no values. To ensure that the results are synced and that undefined values are not parsed, specify a start date when fetching data so that the *Chart* toolip shows correct values. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1906304[*BZ#1906304*]. 
+* Parsing undefined values results in a not a number (NaN) exception and the *Chart* tooltip shows a box with no values. To ensure that the results are synced and that undefined values are not parsed, specify a start date when fetching data so that the *Chart* toolip shows correct values. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1906304[*BZ#1906304*].
+
+* In {product-title} 4.3 and 4.4, if the user has the console open in multiple tabs, some sidebar links in the *Developer perspective* do not directly link to the project, and there is an unexpected shift in the selected project. This will be resolved in a future release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1839101[*BZ#1839101*])
```

Comment:
What's a _shift in the selected project_?
Should this be a Known Issue rather than a Bug Fix?